### ISSUE TITLE
DRM prime video wont play in brave but it works for other browsers.

### ISSUE THREAD

**Issue by meistfx** [brave/brave-browser#17286]
![unknown](https://user-images.githubusercontent.com/88292541/127788175-f5a4444f-f1a2-4e54-9f2f-429bc123a1fd.png)

## Description 

I can not view amazon prime video with brave I get errors usually 7301. I can how ever view the video with other browsers this is only related to brave. I tried to disable the drm in settings and allow the drm plugin/extention reinstalling it does not fix. I've tried to use different DNS providers. Disabled windows firewall and defender. disabled firewall in router even. Tried using a new profile in brave and even downloaded brave nightly and created a fresh install and it does not matter I currently still get the 7301 error for prime video can not play. 

## Steps to Reproduce

   1.  Open any brave browser
   2.  Go to Amazon, test video: https://www.amazon.com/When-Stranger-Calls-Lance-Henriksen/dp/B0094M140A/
   3.  click play now on a prime video.

## Actual result:

![1](https://user-images.githubusercontent.com/88292541/127786756-b540b51c-24a0-4483-bac8-ddd8a289094d.png)
![2](https://user-images.githubusercontent.com/88292541/127786758-fe83c3f2-3fae-44ef-9c11-2c7ce673f352.png)
![3](https://user-images.githubusercontent.com/88292541/127786759-12f5cc02-a76d-4a22-9bd5-26df09d546d7.png)

## Expected result:
Not sure what you mean by expected results other than I expect a browser as good as people say it to be saying brave is a good browser but can't even play movies I'm paying for yeah that is what I expected. the video to play and work in a browser people claim to be good. 

## Reproduces how often: 

happens every single time I try to play prime videos or if I had to guess anything to do with DRM.

## Brave version (brave://version info)

Brave |1.27.109 Chromium: 92.0.4515.115 (Official Build) (64-bit)
Revision | 48cb2f4029b84b003719740a6cf9ca73f374a857-refs/branch-heads/4515_105@{#4}

and

Brave | 1.29.42 Chromium: 92.0.4515.115 (Official Build) nightly (64-bit)
Revision | 48cb2f4029b84b003719740a6cf9ca73f374a857-refs/branch-heads/4515_105@{#4}

## Version/Channel Information:

- Can you reproduce this issue with the current release? yes
- Can you reproduce this issue with the beta channel? yes
- Can you reproduce this issue with the nightly channel? yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? no
- Does the issue resolve itself when disabling Brave Rewards? no
- Is the issue reproducible on the latest version of Chrome? no google's chrome works fine and so does edge. firefox works great too!

## Miscellaneous Information:

I was told to include console.log 

```
fls-na.amazon.com/1/action-impressions/1/OP/aiv-web-player/action/HTML5Player:.AppLoadCacheMiss@v=1?marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=AIVWebPlayer&session=13963574639258436:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OP/aiv-web-player/action/HTML5Player:.AppLoadTime@v=278:u=ms?marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=AIVWebPlayer&session=9543734203033367:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/aiv-web-player/action/HTML5Player:.LanguageDownload.Time.en-US@v=0:u=ms?instance=f22dbddb-ef2c-48c5-8876-bed0d47594fd&marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=AIVWebPlayer&session=89fdff73-eded-46eb-9b72-ea6cbeeea7d7&client=Chrome:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/aiv-web-player/action/HTML5Player:.LanguageDownload.Time.en-US@v=0:u=ms?instance=f22dbddb-ef2c-48c5-8876-bed0d47594fd&marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=AIVWebPlayer&session=89fdff73-eded-46eb-9b72-ea6cbeeea7d7:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/aiv-web-player/action/HTML5Player:.LanguageDownloadCacheHit@v=1?instance=f22dbddb-ef2c-48c5-8876-bed0d47594fd&marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=AIVWebPlayer&session=89fdff73-eded-46eb-9b72-ea6cbeeea7d7&client=Chrome:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/aiv-web-player/action/HTML5Player:.LanguageDownloadCacheHit@v=1?instance=f22dbddb-ef2c-48c5-8876-bed0d47594fd&marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=AIVWebPlayer&session=89fdff73-eded-46eb-9b72-ea6cbeeea7d7:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
unagi-na.amazon.com/1/events/com.amazon.csm.nexusclient.prod:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
/empty.gif?1627611663417:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/aiv-web-player/action/HTML5Player:.XpLoaderJavaScriptDownloadTime@v=955:u=ms?instance=f22dbddb-ef2c-48c5-8876-bed0d47594fd&marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=AIVWebPlayer&session=89fdff73-eded-46eb-9b72-ea6cbeeea7d7&client=Chrome:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/aiv-web-player/action/HTML5Player:.XpLoaderJavaScriptDownloadTime@v=955:u=ms?instance=f22dbddb-ef2c-48c5-8876-bed0d47594fd&marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=AIVWebPlayer&session=89fdff73-eded-46eb-9b72-ea6cbeeea7d7:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/aiv-web-player/action/HTML5Player:.XpLoaderJavaScriptDownloadTime.cloudfront@v=955:u=ms?instance=f22dbddb-ef2c-48c5-8876-bed0d47594fd&marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=AIVWebPlayer&session=89fdff73-eded-46eb-9b72-ea6cbeeea7d7&client=Chrome:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/aiv-web-player/action/HTML5Player:.XpLoaderJavaScriptDownloadTime.cloudfront@v=955:u=ms?instance=f22dbddb-ef2c-48c5-8876-bed0d47594fd&marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=AIVWebPlayer&session=89fdff73-eded-46eb-9b72-ea6cbeeea7d7:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/aiv-web-player/action/HTML5Player:.XpLoaderJavaScriptCacheMiss@v=1?instance=f22dbddb-ef2c-48c5-8876-bed0d47594fd&marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=AIVWebPlayer&session=89fdff73-eded-46eb-9b72-ea6cbeeea7d7&client=Chrome:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/aiv-web-player/action/HTML5Player:.XpLoaderJavaScriptCacheMiss@v=1?instance=f22dbddb-ef2c-48c5-8876-bed0d47594fd&marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=AIVWebPlayer&session=89fdff73-eded-46eb-9b72-ea6cbeeea7d7:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
unagi.amazon.com/1/events/com.amazon.csm.csa.prod:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
unagi.amazon.com/1/events/com.amazon.csm.csa.prod:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
unagi.amazon.com/1/events/com.amazon.csm.csa.prod:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/aiv-web-player/action/HTML5Player:.DelegatorInitializeTime@v=1347:u=ms?instance=f22dbddb-ef2c-48c5-8876-bed0d47594fd&marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=AIVWebPlayer&session=89fdff73-eded-46eb-9b72-ea6cbeeea7d7&client=Chrome:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/aiv-web-player/action/HTML5Player:.DelegatorInitializeTime@v=1347:u=ms?instance=f22dbddb-ef2c-48c5-8876-bed0d47594fd&marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=AIVWebPlayer&session=89fdff73-eded-46eb-9b72-ea6cbeeea7d7:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
c.amazon-adsystem.com/bao-csm/forensics/a9-tq-forensics-incremental.min.js:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OP/aiv-web-player/action/HTML5Player:.XpJavaScriptDownloadTime@v=435:u=ms?marketplace=us&marketplaceId=ATVPDKIKX0DER&service=AIVWebPlayer&session=9781873676005648&requestId=14137501266030994:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OP/aiv-web-player/action/HTML5Player:.XpJavaScriptDownloadTime@v=435:u=ms?marketplace=us&marketplaceId=ATVPDKIKX0DER&service=AIVWebPlayer&session=9781873676005648&instance=f22dbddb-ef2c-48c5-8876-bed0d47594fd&requestId=041325326565474274:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OP/aiv-web-player/action/HTML5Player:.XpJavaScriptDownloadTime.cloudfront@v=435:u=ms?marketplace=us&marketplaceId=ATVPDKIKX0DER&service=AIVWebPlayer&session=9781873676005648&requestId=42616394336152386:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OP/aiv-web-player/action/HTML5Player:.XpJavaScriptDownloadTime.cloudfront@v=435:u=ms?marketplace=us&marketplaceId=ATVPDKIKX0DER&service=AIVWebPlayer&session=9781873676005648&instance=f22dbddb-ef2c-48c5-8876-bed0d47594fd&requestId=2469709332582397:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/dv-xp-player/action/XpPlayer:.ServiceRequest.DIS.RequestLatencyTime@v=212:u=ms/XpPlayer:.ServiceRequest.DIS.SuccessfulRequestAtAttempt@v=0?marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=ATVXpPlayer&session=82f6f746-e726-487a-a6f1-b1b16fb4cdf9&host=XP:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/dv-xp-player/action/XpPlayer:.ServiceRequest.DIS.RequestLatencyTime@v=212:u=ms/XpPlayer:.ServiceRequest.DIS.SuccessfulRequestAtAttempt@v=0?marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=ATVXpPlayer&session=82f6f746-e726-487a-a6f1-b1b16fb4cdf9&host=AOAGZA014O5RE:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/dv-xp-player/action/XpPlayer:.ServiceRequest.DIS.RequestLatencyTime@v=212:u=ms/XpPlayer:.ServiceRequest.DIS.SuccessfulRequestAtAttempt@v=0?marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=ATVXpPlayer&session=82f6f746-e726-487a-a6f1-b1b16fb4cdf9&host=AOAGZA014O5RE&client=Unknown:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
ipv6.unagi-na.amazon.com/1/events/com.amazon.eel.pv.xpplayer.prod:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
unagi-na.amazon.com/1/events/com.amazon.eel.pv.xpplayer.prod:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/batch/1/OE/:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/aiv-web-player/action/HTML5Player:.BorgMaturityRatingTime@v=192.20000000018626?marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=AIVWebPlayer&session=0506273e-0253-4807-bf7d-482104800c72&instance=f22dbddb-ef2c-48c5-8876-bed0d47594fd:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/aiv-web-player/action/HTML5Player:.BorgMaturityRatingTime@v=192.20000000018626?marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=AIVWebPlayer&session=0506273e-0253-4807-bf7d-482104800c72&instance=f22dbddb-ef2c-48c5-8876-bed0d47594fd&client=Chrome:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/aiv-web-player/action/HTML5Player:.BorgMaturityRatingTime@v=192.20000000018626?marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=AIVWebPlayer&session=0506273e-0253-4807-bf7d-482104800c72&host=AOAGZA014O5RE&instance=f22dbddb-ef2c-48c5-8876-bed0d47594fd:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/aiv-web-player/action/HTML5Player:.BorgMaturityRatingTime@v=192.20000000018626?marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=AIVWebPlayer&session=0506273e-0253-4807-bf7d-482104800c72&host=AOAGZA014O5RE&instance=f22dbddb-ef2c-48c5-8876-bed0d47594fd&client=Chrome:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/dv-xp-player/action/XpPlayer:.ServiceRequest.Weblabs.RequestLatencyTime@v=78.59999999962747:u=ms/XpPlayer:.InitializeCalled@v=1/XpPlayer:.LoadCalled@v=1/XpPlayer:.ServiceRequest.PRS.PrimaryResources.RequestLatencyTime@v=277.90000000037253:u=ms/XpPlayer:.Manifest.DASH.Download.TransferTime@v=0.20000000018626451:u=ms/XpPlayer:.Manifest.DASH.Download.Latency@v=179.1000000005588:u=ms/XpPlayer:.Manifest.DASH.Download.Throughput@v=989358.6168391682/XpPlayer:.Manifest.DASH.Parse.Time@v=4:u=ms/XpPlayer:.Filter.MediaCapabilities.Time@v=1:u=ms/XpPlayer:.Asset.VastVmap.ParseTime@v=1:u=ms/XpPlayer:.ServiceRequest.GetSections.RequestLatencyTime@v=257.20000000018626:u=ms/XpPlayer:.ServiceRequest.PRS.License.RequestLatencyTime@v=146.90000000037253:u=ms/XpPlayer:.ServiceRequest.PRS.License.RequestLatencyTime@v=136.1000000005588:u=ms/XpPlayer:.ServiceRequest.PRS.License.RequestLatencyTime@v=222.69999999925494:u=ms/XpPlayer:.ServiceRequest.PRS.License.Error@v=1/XpPlayer:.FatalError@v=1?marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=ATVXpPlayer&session=e3d636e6-24ed-4e14-9bae-fb0973f01293&host=XP:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/dv-xp-player/action/XpPlayer:.ServiceRequest.Weblabs.RequestLatencyTime@v=78.59999999962747:u=ms/XpPlayer:.InitializeCalled@v=1/XpPlayer:.LoadCalled@v=1/XpPlayer:.ServiceRequest.PRS.PrimaryResources.RequestLatencyTime@v=277.90000000037253:u=ms/XpPlayer:.Manifest.DASH.Download.TransferTime@v=0.20000000018626451:u=ms/XpPlayer:.Manifest.DASH.Download.Latency@v=179.1000000005588:u=ms/XpPlayer:.Manifest.DASH.Download.Throughput@v=989358.6168391682/XpPlayer:.Manifest.DASH.Parse.Time@v=4:u=ms/XpPlayer:.Filter.MediaCapabilities.Time@v=1:u=ms/XpPlayer:.Asset.VastVmap.ParseTime@v=1:u=ms/XpPlayer:.ServiceRequest.GetSections.RequestLatencyTime@v=257.20000000018626:u=ms/XpPlayer:.ServiceRequest.PRS.License.RequestLatencyTime@v=146.90000000037253:u=ms/XpPlayer:.ServiceRequest.PRS.License.RequestLatencyTime@v=136.1000000005588:u=ms/XpPlayer:.ServiceRequest.PRS.License.RequestLatencyTime@v=222.69999999925494:u=ms/XpPlayer:.ServiceRequest.PRS.License.Error@v=1/XpPlayer:.FatalError@v=1?marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=ATVXpPlayer&session=e3d636e6-24ed-4e14-9bae-fb0973f01293&host=AOAGZA014O5RE:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/action-impressions/1/OE/dv-xp-player/action/XpPlayer:.ServiceRequest.Weblabs.RequestLatencyTime@v=78.59999999962747:u=ms/XpPlayer:.InitializeCalled@v=1/XpPlayer:.LoadCalled@v=1/XpPlayer:.ServiceRequest.PRS.PrimaryResources.RequestLatencyTime@v=277.90000000037253:u=ms/XpPlayer:.Manifest.DASH.Download.TransferTime@v=0.20000000018626451:u=ms/XpPlayer:.Manifest.DASH.Download.Latency@v=179.1000000005588:u=ms/XpPlayer:.Manifest.DASH.Download.Throughput@v=989358.6168391682/XpPlayer:.Manifest.DASH.Parse.Time@v=4:u=ms/XpPlayer:.Filter.MediaCapabilities.Time@v=1:u=ms/XpPlayer:.Asset.VastVmap.ParseTime@v=1:u=ms/XpPlayer:.ServiceRequest.GetSections.RequestLatencyTime@v=257.20000000018626:u=ms/XpPlayer:.ServiceRequest.PRS.License.RequestLatencyTime@v=146.90000000037253:u=ms/XpPlayer:.ServiceRequest.PRS.License.RequestLatencyTime@v=136.1000000005588:u=ms/XpPlayer:.ServiceRequest.PRS.License.RequestLatencyTime@v=222.69999999925494:u=ms/XpPlayer:.ServiceRequest.PRS.License.Error@v=1/XpPlayer:.FatalError@v=1?marketplace=us&marketplaceId=ATVPDKIKX0DER&requestId=1&service=ATVXpPlayer&session=e3d636e6-24ed-4e14-9bae-fb0973f01293&host=AOAGZA014O5RE&client=Chrome:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
unagi.amazon.com/1/events/com.amazon.csm.csa.prod:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
ipv6.unagi-na.amazon.com/1/events/com.amazon.eel.pv.xpplayer.prod:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
unagi-na.amazon.com/1/events/com.amazon.eel.pv.xpplayer.prod:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
fls-na.amazon.com/1/batch/1/OE/:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
DevTools failed to load source map: Could not load content for http://sourcemaps.aka.amazon.com/1.0.35831.0/sourcemaps/vendors.855d38563da8a61ba5cc.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for http://sourcemaps.aka.amazon.com/1.0.35831.0/sourcemaps/commons.73bae31bc23fddd5e218.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for http://sourcemaps.aka.amazon.com/1.0.35831.0/sourcemaps/payment-state-msg.4c2a3e593d1e883117b7.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for http://sourcemaps.aka.amazon.com/1.0.35831.0/sourcemaps/retail-nav.a90c93f318b59b364248.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for http://sourcemaps.aka.amazon.com/1.0.35831.0/sourcemaps/detail.93cfd9413c12d851917e.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for http://sourcemaps.aka.amazon.com/1.0.35831.0/sourcemaps/pangaea-banner.d1238bf77c26fc3ec4ed.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for http://sourcemaps.aka.amazon.com/1.0.35831.0/sourcemaps/942.d6c5cbbbccb26f88e24b.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for http://sourcemaps.aka.amazon.com/1.0.35831.0/sourcemaps/detail-hero-video.a3f489b5822aece5f2cd.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for http://sourcemaps.aka.amazon.com/1.0.35831.0/sourcemaps/detail-purchase.7bb4a58aa627b57e6d79.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for http://sourcemaps.aka.amazon.com/1.0.35831.0/sourcemaps/detail-entitlement.7ec615d3aca2215fba96.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for http://sourcemaps.aka.amazon.com/1.0.35831.0/sourcemaps/detail-hover.084492af61c0007991a4.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
31YXrY93hfL.js:19 POST https://fls-na.amazon.com/1/batch/1/OE/ net::ERR_BLOCKED_BY_CLIENT
send @ 31YXrY93hfL.js:19
x @ 31YXrY93hfL.js:7
z @ 31YXrY93hfL.js:7
ref=atv_dl_rdr?autoplay=1:2326 POST https://unagi-na.amazon.com/1/events/com.amazon.csm.nexusclient.prod net::ERR_BLOCKED_BY_CLIENT
send @ ref=atv_dl_rdr?autoplay=1:2326
A @ ref=atv_dl_rdr?autoplay=1:2328
m @ ref=atv_dl_rdr?autoplay=1:2328
31YXrY93hfL.js:19 POST https://fls-na.amazon.com/1/batch/1/OE/ net::ERR_BLOCKED_BY_CLIENT
send @ 31YXrY93hfL.js:19
x @ 31YXrY93hfL.js:7
z @ 31YXrY93hfL.js:7
setTimeout (async)
C @ 31YXrY93hfL.js:8
K @ 31YXrY93hfL.js:9
f @ ref=atv_dl_rdr?autoplay=1:2298
A @ ref=atv_dl_rdr?autoplay=1:2299
(anonymous) @ ref=atv_dl_rdr?autoplay=1:2299
setTimeout (async)
y @ ref=atv_dl_rdr?autoplay=1:2299
A @ ref=atv_dl_rdr?autoplay=1:2299
u @ ref=atv_dl_rdr?autoplay=1:2299
z @ 31YXrY93hfL.js:7
```

**Comment by ryanbr** [comment]
I was able to play it without issue, Just needed to VPN into the US. Try disabling widevine, restart, and then re-enabling widvine (in `brave://settings/extensions` ) @meistfx

**Comment by rebron** [comment]
@meistfx Were you able to toggle off/on the Widevine setting to get the Widevine re-installed for you. You'll need widevine installed to view these videos. If you can check brave://components/ , see if Widevine Content Decryption Module is there.

**Comment by bsclifton** [comment]
Pic of the setting (mentioned above:
![image](https://user-images.githubusercontent.com/4733304/129416974-d10c4e3b-ad88-43a1-bb41-c5e7d6b27356.png)

It's found under brave://settings/extensions. You might toggle off and then back on, so it downloads. You can then see it in brave://components
![image](https://user-images.githubusercontent.com/4733304/129417202-74436839-600c-43e4-bced-f911622dc6dd.png)

**Comment by hansjhoffman** [comment]
I've had the same issue for about 4 weeks (using version 1.28.105 and the previous version). I've tried the suggest widevine toggle but it doesn't work. I'm surprised more people aren't experiencing & reporting this issue.

**Comment by bsclifton** [comment]
@hansjhoffman what OS (and version) are you running? Will be looking more into this

**Comment by ryanbr** [comment]
Tested https://www.amazon.com/gp/video/detail/B0875THX53/ref=atv_hm_hom_3_c_dVClNC_2_2 in a clean profile, no issues playing the video. In one instance, It did detect if I was running a VPN (and stopped playback because of it). @meistfx @hansjhoffman

**Comment by kjozwiak** [comment]
Seems to be working for me as well on `Win 10 x64` using the following build:

```
Brave | 1.30.23 Chromium: 92.0.4515.131 (Official Build) nightly (64-bit)
-- | --
Revision | 6b8d6c56ce21e38a72f7c4becb5abc1fa5134f29-refs/branch-heads/4515@{#1933}
OS | Windows 10 OS Version 2009 (Build 19042.1165)
```

I also tried it on a new profile using `1.28.105 Chromium: 92.0.4515.131` which is the latest release and went through the following:

* visited https://www.amazon.com/gp/video/detail/B0875THX53/ref=atv_hm_hom_3_c_dVClNC_2_2
* installed Widevine via the prompt/modal
* logged into Amazon Canada
* played the video mentioned above

Seems to be working for me on `Win 10 x64`.

**Comment by aquadzn** [comment]
M1
Brave Version 1.29.76 Chromium: 93.0.4577.58 (Official Build) (arm64)
Widevine on

Not working for me, no error code. Works with Safari
![image](https://user-images.githubusercontent.com/46140458/131515842-63c5772b-326c-4a63-8858-eaf8620e5808.png)

**Comment by aquadzn** [comment]
> @hansjhoffman and @aquadzn Can you update to Brave 1.31.87 and let us know if you're still running into this issue? Also, what countries are you in?
> 
> I'm on M1 and looks like the video from STR is running fine. Wondering if there's some region specific issue here.

Hello, I updated brave and it's working thanks

**Comment by kneekoo** [comment]
This is a problem for me, and disabling and re-enabling the extension didn't help. As a matter of fact, it was worse once I restarted the browser from the extensions page - with Netflix no longer working either.

Once toggled, Brave asks if you want to restart it. So I disabled Widevine and restarted, then re-enabled it and restarted Brave. This is where Netflix also stopped working. So I disabled it again and restarted the browser, refreshed Netflix and Brave gave me a sort of a notification/dialog below the address bar, telling me about Netflix requiring encrypted content, and asking if I want to enable it. So I enabled it from that dialog and it worked after another browser restart.

However, after all these attempts, Amazon Prime Video still won't work, with the same error message (/dv/error/7031).
